eventfiles/class_tests/event_file_views_tests.py

```diff
@@ -37,13 +54,77 @@ def test_render_files_template(self, client, sign_in):
         "event doesnt exist",
         "user not a participant in the event",
     ])
-    def test_invalid_event_id_in_file_url(self, client, sign_in, url_path, status_code):
+    def test_invalid_event_id_in_file_url(self, client, sign_in_with_creator, url_path, status_code):
         response = client.get(url_path)
         assert response.status_code == status_code
 
-    def test_unauthorized_user_redirected_to_main_page(self, client):
+    def test_unauthorized_user_redirected_to_login_page(self, client):
         response = client.get(f'{FILE_CREATION_URL}/1')
         assert response.status_code == 302
         response = client.get(response.url)
         assert response.status_code == 200
-        assertTemplateUsed(response, 'main/index.html')
+        assertTemplateUsed(response, 'login/register_login.html')
+
+    def test_all_event_files_are_shown_and_downloadable(self, sign_in_with_user2, client):
+        response = client.get(reverse("event_files", args=[EVENT_ID]))
+        char_content = response.content.decode(response.charset)
```

Comment:
I think you can use:
```suggestion
        char_content = get_char_content_from_eventfile_html(self, client)
```